pkg/crds/enterprise/crd.projectcalico.org_felixconfigurations.yaml

```diff
@@ -108,7 +108,7 @@ spec:
                 description: 'BPFExtToServiceConnmark in BPF mode, control a 32bit
                   mark that is set on connections from an external client to a local
                   service. This mark allows us to control how packets of that connection
-                  are routed within the host and how is routing interpreted by RPF
+                  are routed within the host and how is routing intepreted by RPF
```

Comment:
Hehe, this went from one typo to another. But it is not a request for a fix in your pr :)